app/models/message_custom_prompt.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class MessageCustomPrompt < ActiveRecord::Base
+  # belongs_to chat message but going to avoid the cross dependency for now
+end
+
+# == Schema Information
+#
+# Table name: message_custom_prompts
+#
+#  id            :bigint           not null, primary key
+#  message_id    :bigint           not null
+#  custom_prompt :json             not null
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#
+# Indexes
+#
+#  index_message_custom_prompts_on_message_id  (message_id) UNIQUE
+#
```

Comment:
Maybe we prefix with `chat_` here. To simplify things in future?

`self.table_name = "chat_message_custom_prompts"`

Reply:
I can rename no problem ... just tried to leave this even with PostCustomPrompt